visuals.Rmd

```diff
@@ -227,6 +227,90 @@ vis_drake_graph(
 )
 ```
 
+## Node Selection
+
+First, we define our plan, adding a custom column named "link".
+
+```{r}
+mtcars_link <- 
+  "https://stat.ethz.ch/R-manual/R-devel/library/datasets/html/mtcars.html"
+iris_link <- 
+  "https://archive.ics.uci.edu/ml/datasets/iris"
+iris3_link <- 
+  "https://www.rdocumentation.org/packages/datasets/versions/3.6.1/topics/iris"
+
+plan <- drake::drake_plan(
+  mtc = target(
+    mtcars,
+    link = !!mtcars_link
+  ),
+  ir = target(
+    iris,
+    link = !!iris_link
+  ),
+  ir3 = target(
+    modify_iris3(iris3, number),
+    transform = map(number = !!c(1:3), .tag_in = cluster_id ),
+    link = !!iris3_link
+  ),
+  trace = TRUE
+)
+```
+```{r}
+config <- drake::drake_config(plan)
+unique_stems <- unique(plan$cluster_id)
+```
+
+
+### Perform the default action on select
+
+By supplying `on_select = TRUE`, `vis_drake_graph()` knows to treat values in
+the column given by `on_select_col` as hyperlinks.
+
+```{r}
+drake::vis_drake_graph(config, clusters = unique_stems, group = "cluster_id",
```

Comment:
Could we indent this code to align with the general style? Also, we do not need `drake::` since `drake` is already loaded.

```{r}
vis_drake_graph(
  config,
  clusters = unique_stems,
  group = "cluster_id",
  hover = FALSE,
  on_select_col = "link",
  on_select = TRUE
)


```